Makefile changes for ONIE patch

Copyright (C) 2015 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Minor Makefile changes for ONIE build environment.

diff --git a/Makefile b/Makefile
index a726773..7d2567c 100644
--- a/Makefile
+++ b/Makefile
@@ -4,6 +4,10 @@
 OPT=-O2
 CFLAGS=$(OPT) -Wall -W -Wno-parentheses -Wstrict-prototypes -Wmissing-prototypes

+# ONIE build environment changes
+CFLAGS  += $(ONIE_CFLAGS)
+LDFLAGS += $(ONIE_LDFLAGS)
+
 VERSION=3.15.0
 DATE=2026-04-05
